docs/src/pages/[platform]/connected-components/storage/fileuploader/react.mdx

```diff
@@ -9,8 +9,8 @@ FileUploaderComponentsExample, LevelExample, VariationExample, AutoUploadExample
 MaxFileSizeExample, ResumableExample, ShowImagesExample, FileUploaderSuccessExample, PreviewerComponentsExample, TrackerComponentsExample, UploadButtonComponentsExample } from './examples'
 import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
 
-<Alert variation="info" heading="Developer Preview">
-  The File Uploader component is in developer preview! 
+<Alert variation="warning" heading="Developer Preview">
+  FileUploader has exited Dev Preview and was renamed to [StorageManager](storagemanager) with some API changes. Please migrate to the StorageManager component, as the FileUploader component will be removed in a future major release.
```

Comment:
Should the link be `[StorageManager](/storagemanager)`?

Reply:
I think the starting slash would mean that link would go to [domain]/storagemanager, without the slash it is relative to the current path so [domain]/react/connected-components/fileuploader -> [domain]/react/connected-components/storagemanager At least that is my understanding